auditlog/concurrent_writer.go

```diff
@@ -76,13 +70,21 @@ func (cl concurrentWriter) Write(al *Log) error {
 	if err != nil {
 		return err
 	}
-	err = os.WriteFile(filepath, []byte(jsdata), cl.auditFileMode)
+	err = os.WriteFile(filepath, jsdata, cl.auditFileMode)
 	if err != nil {
 		return err
 	}
 	cl.mux.Lock()
 	defer cl.mux.Unlock()
-	cl.auditlogger.Println(str)
+
+	cl.auditlogger.Printf("%s %s - - [%s]", al.Transaction.ClientIP, al.Transaction.HostIP, al.Transaction.Timestamp)
```

Comment:
This is somewhat different than above check, omitting some `"-"`, check if it's OK. I'm wondering if we even need this log in addition to the above `WriteFile`